packages/tsconfig.base.json

```diff
@@ -3,7 +3,7 @@
         /* Visit https://aka.ms/tsconfig to read more about this file */
 
         /* Projects */
-        "incremental": false /* Save .tsbuildinfo files to allow for incremental compilation of projects. */,
+        "incremental": true /* Save .tsbuildinfo files to allow for incremental compilation of projects. */,
```

Comment:
Instead of changing this in the base tsconfig can you set it in the playground one?

Reply:
sure!